AxonDeepSeg/segment.py

```diff
@@ -143,6 +143,24 @@ def segment_images(
 
     path_images = [convert_path(p) for p in path_images]
     path_model = convert_path(path_model)
+
+    available_models = ads.get_existing_models_list()
+    if available_models is None:
+        available_models = []
+    if path_model.stem not in available_models:
```

Comment:
Minor nitpick to preserve the distinction between `None` and `[]` (as you explained to me earlier)

```suggestion
    if not available_models or path_model.stem not in available_models:
```